hr_attendance_management/README.rst

```diff
@@ -46,6 +46,14 @@ To configure this module, go in **Attendances/Configuration/Attendance Settings*
 #. In **Leaves/Configuration/Leave Types** configure if leave types should not deduce from the due hours of
    the employees by setting *Keep due hours*
 
+To configure the cron who update the balance annually , go in **Settings/Technical/Automation/Automated Actions**:
+
+#. In the cron *Compute annual extra hours balance* choose the date you want to apply the process to move the extra hours in the annual balance.
```

Comment:
choose the scheduled date on which you want to apply the process to move the extra hours of employees in their annual balance. This is typically done at the beggining of the year.